exercises/concept/black-jack/.meta/exemplar.py

```diff
@@ -1,23 +1,66 @@
-def number_of_card(card):
-    if card == 'A':
-        return "ace"
-    elif card == 'J' or card == 'Q' or card == 'K':
-        return 10
+def value_of_card(card):
+    """
+
+    :param card: str - given card.
+    :return: int - value of a given card (face card = 10, pip value otherwise).
+    """
+
+    if card in ['J', 'Q', 'K']:
+        value = 10
     else:
-        return card
+        value = int(card)
+    return value
+
+
+def value_of_ace(hand_value):
+    """
 
+    :param hand_value: int - current hand value.
+    :return: int - value of the upcoming ace card (either 1 or 11).
+    """
 
-def number_of_ace(hand):
-    if hand + 11 <= 21:
-        return 11
+    if hand_value + 11 > 21:
+        value = 1
     else:
-        return 1
+        value = 11
+    return value
+
+
+def is_blackjack(hand):
+    """
+
+    :param hand: list - a pair of cards in hand.
+    :return: bool - if the hand is a blackjack (two cards worth 21).
+    """
 
+    blackjack = False
+    if 'A' in hand:
+        if any(ten_card in hand for ten_card in ['10', 'J', 'Q', 'K']):
```

Comment:
Was `any()` introduced in an earlier concept? If not, should it be avoided?

Reply:
The [exercise design doc](https://github.com/exercism/python/blob/main/exercises/concept/black-jack/.meta/design.md) doesn't call out `any()` as something to go over, no.  But we largely have avoided making concept exercises about `built-ins`, favoring introducing them inside of or along with other concepts.

I'm open to using `any()` in this exercise, along with `all()`. We just need to make sure they are explained correctly.  
It makes sense to introduce them _somewhere_ between `numbers`, `comparisons`, `conditionals` and `bools`.  But that will also mean that we would need to set `bools` as a prerequisite for this exercise, which may complicate things a bit.

It might make more sense to introduce them in a follow-on exercise to `bools`, where we talk more about how to combine comparisons, conditionals and Boolean logic.

@IsaacG @pranasziaukas  Thoughts on this?